src/ui/window_list.c

```diff
@@ -1247,23 +1180,19 @@ wins_get_next_unread(void)
 
         // test if window has unread messages
         if (win_unread(window) > 0) {
-            g_list_free(values);
             return window;
         }
 
         curr = g_list_next(curr);
     }
 
-    g_list_free(values);
     return NULL;
 }
 
 ProfWin*
 wins_get_next_attention(void)
 {
     // get and sort win nums
-    GList* values = g_hash_table_get_values(windows);
-    values = g_list_sort(values, _wins_cmp_num);
```

Comment:
huh, having a second look at this, this didn't make sense!?

values->data is a `ProfWin*`, why would one want to sort that?

Reply:
@DebXWoody @StefanKropp  maybe knows what he tried to do here.